### PR TITLE
various faq edits

### PR DESCRIPTION
**Pull request by Goober5000**
Some edits, clarifications, and wordsmithing.

### CODE REVIEW

File: communityfaq.json [hunk 1]
```diff
@@ -3,29 +3,29 @@
...
             },
             {
                 "Question":"Why is this launcher called Knossos?",
-                "Answer":"Knossos is the name of something plot related in the Freespace games."
+                "Answer":"Knossos is the name of something plot related in the FreeSpace games."
             },
             {
                 "Question":"What happened to old Knossos?",
-                "Answer":"There were some fundamental issues in the manner it was implemented.  In the end, ShivanSPS created Knossos.NET (KNet for short) to replace it as a project to learn Avalonia UI and provide a new launcher for us."
+                "Answer":"The old version of Knossos was not designed to handle a large number of mods and was also difficult to maintain.  ShivanSPS created Knossos.NET (KNet for short) to fix some of the shortcomings of the original and to learn Avalonia UI."
```
Comment (JohnAFernandez) [inline communityfaq.json:24]: There were more issues with the original Knossos than just the number of mods.  There were issues with how it used async code, as well.  @notimaginative has some horror stories from trying to fix Knossos.  That's why I wasn't specific.

**Review by Shivansps — APPROVED** [review]
Looks good to me

File: communityfaq.json [hunk 2]
```diff
@@ -75,35 +75,35 @@
...
-                "Question":"What is qtFred?",
-                "Answer":"qtFred is an in-development rewrite of Fred using the cross-platform qt framework.  It is about halfway complete."
+                "Question":"What is qtFRED?",
+                "Answer":"qtFRED is an in-development rewrite of FRED using the cross-platform Qt framework.  It is about halfway complete."
             },
             {
-                "Question":"What are sexps?",
-                "Answer":"Sexps are symbolic expressions, the original mission and campaign action language.  Sexps are lisp-like and very powerful. You can also define custom variables and containers with them."
+                "Question":"What are SEXPs?",
+                "Answer":"SEXPs, or s-exps (symbolic expressions), are the building blocks for the gameplay logic in FreeSpace missions and campaigns.  SEXPs are adapted from the language Lisp and are very powerful.  Numerous SEXP operators have been added to the original basic collection implemented by Volition."
```
Comment (JohnAFernandez) [inline communityfaq.json:90]:
```suggestion
                "Answer":"SEXPs, or s-exps are symbolic expressions. They are the building blocks for the gameplay logic in FreeSpace missions and campaigns.  SEXPs are adapted from the language Lisp and are very powerful.  Numerous SEXP operators have been added to the original basic collection implemented by Volition."
```

File: communityfaq.json [hunk 3]
```diff
@@ -75,35 +75,35 @@
...
             },
             {
-                "Question":"What is FSO LUA?",
-                "Answer":"Our engine’s scripts are written in Lua, and it can be used to greatly increase the capabilities of the engine without waiting for native code support."
+                "Question":"What is Lua?",
```
Comment (JohnAFernandez) [inline communityfaq.json]:
```suggestion
                "Question":"What is Lua, and how does FSO use it?",
```

**Review by JohnAFernandez — CHANGES_REQUESTED** [review]

File: communityfaq.json (hunk 2 above)
Comment (Goober5000) [reply communityfaq.json:90]: This suggestion seems a bit tautological; "s-exp" is literally the abbreviation for "symbolic expression".  IMO it makes sense to put the abbreviation expansion in parentheses and use the "are" linking verb to introduce the full definition.

File: communityfaq.json (hunk 1 above)
Comment (Goober5000) [reply communityfaq.json:24]: Well yeah, I didn't intend for this to be an exhaustive description.  My intent here was to provide one specific example (scaling to a large number of mods, which taylor and I have discussed previously) and one general example (being difficult to maintain, which encompasses not only the choice of language but also a whole bunch of issues, including the async code).  This provides readers with an idea of the difficulties without getting too bogged down in the details.

File: communityfaq.json (hunk 2 above)
Comment (JohnAFernandez) [reply communityfaq.json:90]: Parentheses are not going to add clarity or ease of understanding to the target audience of an FAQ.

File: communityfaq.json (hunk 1 above)
Comment (JohnAFernandez) [reply communityfaq.json:24]: You're adding details to avoid getting bogged down in details?  If they want an example, they can just ask us.

**Review by JohnAFernandez — APPROVED** [review]

**Pull request merged**